src/ontology/edits/oeo-shared.omn

```diff
@@ -1253,24 +1285,30 @@ Class: OEO_00020154
         <http://purl.obolibrary.org/obo/IAO_0000115> "Emission certificate price is a monetary price of an emission certificate that allows the emission of a certain emission value.",
         <http://purl.obolibrary.org/obo/IAO_0000118> "CO2 price",
         <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/866
-pull request: https://github.com/OpenEnergyPlatform/ontology/pull/956",
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/956
+add relation to emission certificate
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1034",
         rdfs:label "emission certificate price"
     
     SubClassOf: 
-        OEO_00040003
+        OEO_00040003,
+        OEO_00020179 some OEO_00020063
     
     
 Class: OEO_00020155
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "Social cost of carbon is the cost the society bears for an emission value.",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Social cost of carbon emission is the cost the society bears for a greenhouse gas emission value.",
         <http://purl.obolibrary.org/obo/IAO_0000118> "social cost of CO2",
         <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/866
-pull request: https://github.com/OpenEnergyPlatform/ontology/pull/956",
-        rdfs:label "social cost of carbon"
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/956
+add axiom to emission value, add new parent class and relable
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1034",
+        rdfs:label "social cost of carbon emission"
     
     SubClassOf: 
-        OEO_00040009,
+        OEO_00020181,
+        OEO_00020179 some OEO_00140081,
```

Comment:
This should be `'economic value of' some 'greenhouse gas emission value'`

Reply:
This requires moving a bunch of classes to oeo-shared. Let's wait for #1036 here.

Reply:
Ok.